tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -2345,6 +2345,30 @@ def output_types(self):
     return self._output_types
 
 
+class MatchingFilesDataset(Dataset):
+  """A `Dataset` that list the files according to the input patterns."""
+
+  def __init__(self, patterns):
+    super(MatchingFilesDataset, self).__init__()
+    self._patterns = ops.convert_to_tensor(
+      patterns, dtype=dtypes.string, name="patterns")
```

Comment:
the indentation on this line in incorrect (please follow https://www.tensorflow.org/community/style_guide)